fdbserver/ClusterController.actor.cpp

```diff
@@ -1351,7 +1348,6 @@ class ClusterControllerData {
 	Counter getWorkersRequests;
 	Counter getClientWorkersRequests;
 	Counter registerMasterRequests;
-	Counter getServerDBInfoRequests;
```

Comment:
What is the reason for removing this metric?

Reply:
At one point this endpoint was removed completely, however I had to bring it back for the tester. Because the tester is the only process which uses then endpoint it still seems like it might be the right choice to avoid logging it.

Reply:
Fair enough. Can we comment somewhere that it's only used by testers?